public/js/index.js

```diff
@@ -2786,6 +2786,7 @@ function updateViewInner () {
   renderTOC(ui.area.markdown)
   generateToc('ui-toc')
   generateToc('ui-toc-affix')
+  autoLinkify(ui.area.markdown)
```

Comment:
Hmm, why here calls `autoLinkify` again?

Reply:
Because ToC adds those `tip<number>` ids. Autolinkify in first place adds ids to all headers. At this point in time the `tip` ids, don't exist. So the anchors will link to `#`. Calling it twice is actually a dirty hack but fixes the problem of anchors that point into nowhere and makes emtpy header linkable. (using the ids, generated by ToC)

Reply:
That's ok.